File: task-03/C/SubTask_2.c
#include <stdio.h>

int main() {
    FILE *input,*output;
    input=fopen("input.txt","r");
    output=fopen("output.txt","w");
    char ch;
    while((ch=fgetc(input))!=EOF){
        fputc(ch,output);
    }
    fclose(input);
    fclose(output);
    return 0;
}

File: task-03/Java/SubTask_2.java
import java.io.File;
import java.io.FileNotFoundException;
import java.io.PrintWriter;
import java.util.Scanner;

public class SubTask_2 {
    public static void main(String[] args) {
        File input=new File("input.txt");
        File output=new File("output.txt");
        try {
            Scanner sc=new Scanner(input);
            PrintWriter write=new PrintWriter(output);
            while(sc.hasNextLine()){
                write.println(sc.nextLine());
            }
            sc.close();
            write.close();
        } 
        catch(FileNotFoundException e) {
            e.printStackTrace();
        }
    }
}
